### PR TITLE
parameter r

### LINKED ISSUE #12: Parameter -r
Causes the comparator to be wrapped into a function applying the -r effect on the original comparator's result.

### PR DESCRIPTION
**Pull request by justinmueller05082000**
fixes #12

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -20,10 +20,10 @@ func main() {
 	leadingBlanksParameter := flag.Bool("b", false, "Ignore leading blanks")
 	ignoreCaseParameter := flag.Bool("f", false, "Fold lower case to upper case characters")
 	randomSortingParameter := flag.Bool("R", false, "shuffle, but group identical keys.")
+	reversePrintingParameter := flag.Bool("r", false, "Print result in reverse order")
 	flag.Parse()
 
 	var comparator func(a, b []byte) int
-
```
Comment (Al2Klimov) [inline main.go]: Not part of the task.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [x] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD master)..HEAD^)`
* [x] `git commit --amend --no-edit`
* [x] `git push -f origin feature/parameter-r-12`

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Write a test protocol.

**Comment by justinmueller05082000** [comment]
**Test protocol:**

**toSort.txt:**
```
5
4
6
7
9
0
2
1
5
7
8
4
3
```

**GO CODE:**
**Terminal:** `go run . -r toSort.txt` 
**Output:**
```
9
8
7
7
6
5
5
4
4
3
2
1
0
```


**SORT COMMAND:**
**Terminal:** `sort -r toSort.txt` 
**Output:**
```
9
8
7
7
6
5
5
4
4
3
2
1
0
```


**Conclusion:**
Both results are sorted and printed equally in the right and reverse order.

**Pull request merged**